### ISSUE TITLE
[Package Issue]: Python.Python.3.12: `python3` moniker causes conflict

### ISSUE THREAD

**Issue by michaelmhoffman** [microsoft/winget-pkgs#132377]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

The `python3` moniker should refer to the latest version of Python 3 (as discussed in #67365 and #85695). @Dragon1573 added a helpfully `Python.Python.3.13` package in #131741 in response to #131729. However, it has the `python3` moniker, which it shouldn't until it is the stable version. At that point, the `python3` moniker should be removed from the `Python.Python.3.12` package so it only points to one package.

I suggest:
1. removal of the `python3` moniker from `Python.Python.3.12`
2. addition of a comment around the moniker line in both the `Python.Python.3.12` and `Python.Python.3.13` packages explaining that the moniker should only be in one package which has the latest stable or maintenance release, and that when there is a new stable release, the moniker needs removing from the older package.

### Steps to reproduce

```powershell
winget install --exact --moniker python3
```

### Actual behavior

```console
Multiple packages found matching input criteria. Please refine the input.
Name            Id                 Source
-----------------------------------------
Python 3.11     Python.Python.3.11 winget
Python 3.12     Python.Python.3.12 winget
Python 3.13.0a2 Python.Python.3.13 winget
```

### Expected behavior

Installation of the most recent stable or maintenance version of Python3, `Python3.Python3.12`.

### Environment

```raw
Windows Package Manager v1.6.3482
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.2861
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.21.3482.0
```

**Comment by Dragon1573** [comment]
> The `python3` moniker should refer to the latest version of Python 3.

@michaelmhoffman - When try installing Python with command `winget install -e --moniker python3`, it should always point to the latest General Available version, currently `v3.12`?

I would preffer removing `python3` moniker from `Python.Python.3.11` and `Python.Python.3.13`. I'll perform this change for you. 😄

**Comment by Dragon1573** [comment]
`Moniker: python3` has removed from package identifier `Python.Python.3.11`. I'm working on removing it from package identifier `Python.Python.3.13`.

```text
D:\Workloads> winget search --exact --moniker python3
名称            ID                 版本     匹配             源
-------------------------------------------------------------------
Python 3.12     Python.Python.3.12 3.12.1   Moniker: python3 winget
Python 3.13.0a2 Python.Python.3.13 3.13.0a2 Moniker: python3 winget
```
